tests/trustyai/constants.py

```diff
@@ -0,0 +1,5 @@
+TIMEOUT_1MIN = 60
+TIMEOUT_5MIN = 5 * TIMEOUT_1MIN
+
+TRUSTYAI_SERVICE: str = "trustyai-service"
+MODELMESH_SERVING = "modelmesh-serving"
```

Comment:
This seems like a higher level constant? Or is it something you only use in trusty?

Reply:
Right now it is only used in trusty, but when we need it somewhere else (e.g. wider model serving), we can move it to a more general file.